CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 ### Changed
-- Avoid the print of highest logger levels when the exception is re-thrown. [PR 1187](https://github.com/pgjdbc/pgjdbc/pull/1187)
+- Reduce the severity of the erorr log messages when an exception is re-thrown. We have already logged the
```

Comment:
> We have already logged the error

That is not true.

Actual case is "the error will be thrown to upper level, and it would likely log it/act accordingly", so no need to log it with high severity at pgjdbc side.